app/models/manageiq/providers/amazon/storage_manager/ebs/refresh_parser_inventory_object.rb

```diff
@@ -41,6 +41,10 @@ def parse_volume(volume)
       :base_snapshot         => inventory_collections[:cloud_volume_snapshots].lazy_find(volume['snapshot_id']),
       :availability_zone     => inventory_collections[:availability_zones].lazy_find(volume['availability_zone'])
     }
+
+    link_volume_to_disk(volume_hash, volume['attachments'])
```

Comment:
right, so looking at the OpenStack and a current implementation, seems like ephemeral disks are filled on the CloudManager side and Volumes are added to that on the StorageManager side.

We could probably set it all from the CloudManager side. Or not sure if the AWS even supports ephemeral disks? So local VM disks, if not, we could move it all to storage manager.